pkg/domain/entities/play.go

```diff
@@ -56,6 +56,8 @@ type PlayKubeOptions struct {
 	Start types.OptionalBool
 	// ServiceContainer - creates a service container that is started before and is stopped after all pods.
 	ServiceContainer bool
+	// URL indicates whether to create a yaml file from a URL
+	URL bool
```

Comment:
That's not used. Probably an artifact from an earlier development state?

Reply:
Yes this is from when I was considering using a flag to read the URL before realising the flag was unnecessary.